pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -53,7 +53,7 @@ def query_remote(self, data: dict, modality: str) -> Optional[str]:
         response = self._post(
             f"/modalities/{modality}/query",
             data=data,
-            timeout=config("PIXL_QUERY_TIMEOUT", default=10, cast=float),
+            timeout=config("PIXL_QUERY_TIMEOUT", default=120, cast=float),
```

Comment:
this feels quite long to find out if something exists on the VNA, why the change?

Reply:
Ah good point, I copied from what it is set to in `test/.env`:
https://github.com/UCLH-Foundry/PIXL/blob/2db63312b7aeeea6c38233a2a9ccf9d0f85857ea/test/.env#L4
🤷‍♂️

Reply:
I'll revert it back to

```suggestion
            timeout=config("PIXL_QUERY_TIMEOUT", default=10, cast=float),
```